.github/workflows/spotBugs.yml

```
@@ -0,0 +1,44 @@
+                    distribution: temurin
+                    java-version: 21
+                    cache: gradle
+
+            -   name: Build with Gradle
+                run: ./gradlew clean build -x test
+
+            -   name: Run static code analysis with SpotBugs
+                run: ./gradlew spotbugsMain
+
+            -   name: Upload SpotBugs report to GitHub Actions artifacts
+                uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)